### ISSUE TITLE
Not parsing test cases on Codechef platform

### ISSUE THREAD

**Issue by Ravi1359** [jmerle/competitive-companion#417]
Is this extenstion blocked on [codechef](https://www.codechef.com/) because whenever you try it always shows this error :

![Screenshot (114)](https://github.com/jmerle/competitive-companion/assets/82272263/c22d4934-1092-49e6-a86b-eccb1996b368)

**Comment by jmerle** [comment]
This is a duplicate of #410, for which the fix was already released in version [2.50.0](https://github.com/jmerle/competitive-companion/releases/tag/2.50.0).
